### PR TITLE
Button mapping + commented out deez nuts joke

### PR DESCRIPTION
**Pull request by domoman10001**
Everything said in the title

### CODE REVIEW

File: src/main/cpp/TeleopControl.cpp [hunk 1]
```diff
@@ -116,27 +116,30 @@ void TeleopControl::Initialize()
...
-
-		m_controllerIndex[ARM_GOING_UP]						= ctrlNo;
-		m_buttonIDs[ARM_GOING_UP]							= IDragonGamePad::BUTTON_IDENTIFIER::RIGHT_TRIGGER_PRESSED;
-		m_controllerIndex[ARM_GOING_DOWN]					= ctrlNo;
-		m_buttonIDs[ARM_GOING_DOWN]							= IDragonGamePad::BUTTON_IDENTIFIER::LEFT_TRIGGER_PRESSED;
+		m_controllerIndex[INTAKE_BALLZ] 					= ctrlNo;  
+		m_buttonIDs[INTAKE_BALLZ]	 						= IDragonGamePad::RIGHT_BUMPER;	
+		m_controllerIndex[EXPEL_BALLZ]						= ctrlNo;
+		m_buttonIDs[EXPEL_BALLZ]							= IDragonGamePad::B_BUTTON;
+		m_controllerIndex[DELIVER_BALLZ]					= ctrlNo;
```
Comment (nickfrisch) [inline src/main/cpp/TeleopControl.cpp]: Better button binding names

File: src/main/cpp/TeleopControl.cpp [hunk 2]
```diff
@@ -116,27 +116,30 @@ void TeleopControl::Initialize()
...
-		//m_buttonIDs[ARM_GOING_DOWN]							= IDragonGamePad::POV_180;
-
-		m_controllerIndex[ARM_GOING_UP]						= ctrlNo;
-		m_buttonIDs[ARM_GOING_UP]							= IDragonGamePad::BUTTON_IDENTIFIER::RIGHT_TRIGGER_PRESSED;
-		m_controllerIndex[ARM_GOING_DOWN]					= ctrlNo;
-		m_buttonIDs[ARM_GOING_DOWN]							= IDragonGamePad::BUTTON_IDENTIFIER::LEFT_TRIGGER_PRESSED;
+		m_controllerIndex[INTAKE_BALLZ] 					= ctrlNo;  
+		m_buttonIDs[INTAKE_BALLZ]	 						= IDragonGamePad::RIGHT_BUMPER;	
+		m_controllerIndex[EXPEL_BALLZ]						= ctrlNo;
+		m_buttonIDs[EXPEL_BALLZ]							= IDragonGamePad::B_BUTTON;
```
Comment (nickfrisch) [inline src/main/cpp/TeleopControl.cpp]: Better button binding names

File: src/main/cpp/TeleopControl.cpp [hunk 3]
```diff
@@ -116,27 +116,30 @@ void TeleopControl::Initialize()
...
-		//m_buttonIDs[ARM_GOING_UP]							= IDragonGamePad::POV_0;
-		//m_controllerIndex[ARM_GOING_DOWN]					= ctrlNo;
-		//m_buttonIDs[ARM_GOING_DOWN]							= IDragonGamePad::POV_180;
-
-		m_controllerIndex[ARM_GOING_UP]						= ctrlNo;
-		m_buttonIDs[ARM_GOING_UP]							= IDragonGamePad::BUTTON_IDENTIFIER::RIGHT_TRIGGER_PRESSED;
-		m_controllerIndex[ARM_GOING_DOWN]					= ctrlNo;
-		m_buttonIDs[ARM_GOING_DOWN]							= IDragonGamePad::BUTTON_IDENTIFIER::LEFT_TRIGGER_PRESSED;
+		m_controllerIndex[INTAKE_BALLZ] 					= ctrlNo;  
+		m_buttonIDs[INTAKE_BALLZ]	 						= IDragonGamePad::RIGHT_BUMPER;
```
Comment (nickfrisch) [inline src/main/cpp/TeleopControl.cpp]: Better button binding names

File: src/main/cpp/TeleopControl.cpp (hunk 2 above)
Comment (domoman10001) [reply src/main/cpp/TeleopControl.cpp]: Deal with it because funny :)

File: src/main/cpp/TeleopControl.cpp (hunk 1 above)
Comment (domoman10001) [reply src/main/cpp/TeleopControl.cpp]: Deal with it because funny :)

File: src/main/cpp/TeleopControl.cpp (hunk 3 above)
Comment (domoman10001) [reply src/main/cpp/TeleopControl.cpp]: Deal with it because funny :)

File: src/main/cpp/TeleopControl.cpp (hunk 2 above)
Comment (nickfrisch) [inline src/main/cpp/TeleopControl.cpp]: Do this

**Review by nickfrisch — CHANGES_REQUESTED** [review]

**Review by nickfrisch — APPROVED** [review]

**Pull request merged**